src/controllers/itemdetailpage.js

```diff
@@ -1237,7 +1237,7 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "cardBuild
             view.addEventListener("viewbeforehide", function() {
                 events.off(apiClient, "message", onWebSocketMessage);
                 events.off(playbackManager, "playerchange", onPlayerChange);
-                libraryMenu.setTransparentMenu(false);
+                libraryMenu.setTransparentMenu(true);
```

Comment:
How can this be a fix? According to `git blame` this `setTransparentMenu(false)` was so from the get-go, how can it be broken in 10.4-RC?

Reply:
i think this was broken form the start.

Reply:
If I read the bug correctly, it says that menu started being opaque after 10.3.7... How can it be broken from the beginning?..

Reply:
i will check it on 10.3.7

Reply:
yeah you are correct. This was removed https://github.com/jellyfin/jellyfin-web/pull/325/files